ffc/uflacs/build_uflacs_ir.py

```diff
@@ -594,16 +594,6 @@ def build_uflacs_ir(cell, integral_type, entitytype, integrands, tensor_shape,
                     block_restrictions, block_is_transposed, block_is_uniform, pname)
                 block_is_piecewise = False
 
-            elif block_mode == "scaled":  # TODO: Add mode, block is piecewise but choose not to be premultiplied
-                # Add to contributions:
-                # FI = sum_q weight * f;          generated inside quadloop
-                # B[...] = FI * u * v;            generated after quadloop
-                # A[blockmap] += B[...];          generated after quadloop
-                raise NotImplementedError("scaled block mode not implemented.")
-                # (probably need mostly the same data as
-                # premultiplied, except no P table name or values)
-                block_is_piecewise = False
-
```

Comment:
Could we keep this, please? Some notes how left in UFLACS by Martin are invaluable.